spec/spec_helper.rb

```diff
@@ -113,7 +117,7 @@ def image_path(path, options = {})
 end
 
 VCR.configure do |c|
-  c.cassette_library_dir = Rails.root.join('spec/vcr_cassettes')
+  c.cassette_library_dir = File.join(external_tower_gem_dir, 'spec/vcr_cassettes')
```

Comment:
@durandom @Fryguy This is the part I am not sure about.
This is pulling in *only* cassettes from the external tower repo's.  Haven't been able to find ways to make `vcr` load from multiple cassettes paths because `cassette_library_dir` can be only 1.  Need some advice here.

Good thing is: currently it's ok because `manageiq` specs don't use any other cassettes. And once we split the embedded code, it would be even less to worry about.